File: listas-logpp/lista 2/lista 2 sala/questao 8.py
numero1 = int(input("Digite o primeiro número inteiro: "))
numero2 = int(input("Digite o segundo número inteiro: "))
numero3 = int(input("Digite o terceiro número inteiro: "))

def eh_divisivel_por_2_e_3(numero):
    return numero % 2 == 0 and numero % 3 == 0

numeros_divisiveis = []

for numero in (numero1, numero2, numero3):
    if eh_divisivel_por_2_e_3(numero):
        numeros_divisiveis.append(numero)

if numeros_divisiveis:
    print("Os números divisíveis por 2 e 3 são:", ", ".join(map(str, numeros_divisiveis)))
else:
    print("Nenhum dos números é divisível por 2 e 3.")

File: listas de exercicio/listas 2/lista 2 sala C/questao 8.c
#include <stdio.h>

int main(){
    int N1, N2, N3;
    printf("Digite o primeiro numero:");
    scanf("%d", &N1);
    printf("Digite o segundo numero:");
    scanf("%d", &N2);
    printf("Digite o terceiro numero:");
    scanf("%d", &N3);

    if (N1 % 2==0 && N1 % 3==0){
        printf("%d é divisivel por 2 e 3, ", N1);
    } if(N2 % 2==0 && N2 % 3==0){
         printf("%d é divisivel por 2 e 3, ", N2);
    } if(N3 % 2==0 && N3 % 3==0){
         printf("%d é divisivel por 2 e 3.", N3);
    }
}